### PR TITLE
fix(encryption): encrypt group chat messages

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖

Encrypts and decrypts messages sent to group chats

**Which issue(s) this PR fixes** 🔨

Ap 721

**Additional comments** 🎤

I think testing would just be making sure messages are still sending, unsure though if there is a better way

### CODE REVIEW

File: libraries/Textile/GroupChatManager.ts [hunk 1]
```diff
@@ -91,7 +101,18 @@ export class GroupChatManager {
       groupChatID,
       groupChatQuery,
     )
-    const promises = messages.map<Promise<Message>>(this.decodeMessage)
+    const group = Groups.find((group) => {
+      if (group.address === groupChatID) {
+        return group.encryptionKey
```
Comment (WanderingHogan) [inline libraries/Textile/GroupChatManager.ts:106]: returns the same thing

Comment (WanderingHogan) [reply libraries/Textile/GroupChatManager.ts:106]: Actually, the way it's set up it returns the first object every time

File: libraries/Textile/GroupChatManager.ts [hunk 2]
```diff
@@ -91,7 +101,18 @@ export class GroupChatManager {
...
+      }
+      return undefined
```
Comment (WanderingHogan) [inline libraries/Textile/GroupChatManager.ts:108]: you don't need undefined, but doesn't hurt https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#return_value

**Comment by phillsatellite** [comment]
I noticed img previewer works on group chats now! Looks awesome 💥  should they send as well or is that to be added in future pr? 
Screen Shot 2022-02-16 at 2 34 32 PM

**Comment by Jekrimo** [comment]
@phillsatellite I'm workin on a separate pr for getting our other message types right now that will hopefully be done today!

**Comment by stavares843** [comment]
thanks @Jekrimo 🔨

**Review by phillsatellite — APPROVED** [review]

**Review by iltumio — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**